Makefile

```diff
@@ -1,18 +0,0 @@
-SRC = $(wildcard src/*.js)
```

Comment:
The issues I was having were because of the explicit pathing to mocha and istanbul that are created by using a makefile vs using NPM. You could also use gulp, grunt, or just a plain old node file and run it to get around the issues with using a Makefile. I would say this is the one point you and I just are not going to agree on. 

Are you ok with me leaving the npm commands and adding the Makefile in parallel so that this works on all systems?